lib/common/services/hooks-service.ts

```diff
@@ -121,7 +121,7 @@ export class HooksService implements IHooksService {
 
 				if (invalidArguments.length) {
 					this.$logger.warn(`${hookName} will NOT be executed because it has invalid arguments - ${invalidArguments.join(", ").grey}.`);
```

Comment:
Suggestion: Log the actual file name of the hook with invalid arguments.
Currently the log says: `after-prepare will NOT be executed because it has invalid arguments - platformsData, $usbLiveSyncService.` , but if you have multiple after-prepare hook it would be useful to know which exactly is causing the problem.